models/QtiRunnerInitDataBuilder.php

```diff
@@ -149,13 +166,17 @@ protected function getTestMap(QtiRunnerServiceContext $context)
                     $itemData = $this->qtiRunnerService->getItemData($context, $item->getHref());
 
                     $itemId = $item->getIdentifier();
+                    $score = $itemsStates[$itemId]['score'] ?: [];
```

Comment:
are we sure that `$itemsStates[$itemId]['score']` always exist?

Reply:
Yes, I set this key manually https://github.com/oat-sa/extension-tao-review/pull/18/files#diff-443fde4bf1a7ad77dfc59fc7310a7f95R138
It can be empty, but key exists.

Reply:
for now yes. but array is not a DTO it does not have a contract.
in the future, if someone will modify your code or will reuse getTestMap not carefully we can get a bug.

so I'd say please use self-validated DTO or validate data in array every time.